main.js

```diff
@@ -5,10 +5,9 @@ getElementsByTagName()	  trả về tất cả các phần tử có tên thẻ 
 getElementsByClassName()	trả về tất cả các phần tử có class đã cho.
 ----*/
 
-
 function newElement() {
```

Comment:
Change the name of the function, make it more "action": `createTask`

Reply:
Done